relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -156,9 +147,9 @@ fn extract_user_satisfaction(
             .and_then(|name| config.transaction_thresholds.get(name))
             .unwrap_or(&config.project_threshold);
         if let Some(value) = match threshold.metric {
-            SatisfactionMetric::Duration => {
-                Some(get_duration_millis(start_timestamp, end_timestamp))
-            }
+            SatisfactionMetric::Duration => Some(relay_common::signed_duration_to_millis(
+                end_timestamp - start_timestamp,
```

Comment:
I think this was intentionally a saturating_sub. But this sort of normalization behavior I do think we want to have in a centralized function.

Reply:
ah okay. hmm. it kinda does mean that the duration can underflow now before it is clamped. I am not sure how this is handled by chrono but probably not likely enough

Reply:
The subtraction is `Sub<DateTime<Utc>> for DateTime<Utc>` which is runs [`signed_duration_since`](https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.signed_duration_since) internally. That method is documented with:
> This does not overflow or underflow at all.